src/lib/github.js

```diff
@@ -21,8 +21,4 @@ export const fetchRelease = repo =>
 		.then(d => ({
 			version: d.version || '??',
 			url: d.url || 'https://github.com/preactjs/preact'
-		}))
-		.catch(() => ({
-			url: '#',
-			version: '??'
```

Comment:
Not sure I understand the original idea with this. We have a fallback version and link for a release, so I don't think there's any situation in which we'd want to fall all the way back to displaying `v??` ?

Reply:
I think it was mainly for local dev from the early days

Reply:
Fair enough, admittedly I didn't dig into the blame on that one. Might've predated the fallback version / link.